config.php-init

```diff
@@ -1,22 +0,0 @@
-<?php
-
-return array(
-
-    'database' => array(
-        'user' => '',
-        'pass' => '',
-        'host' => '',
-        'name' => '',
-    ),
-    
-    'routes' => array(
-        '' => 'index/index',
-        'story' => 'story/index',
-        'story/create' => 'story/create',
-        'comment/create' => 'comment/create',
-        'user/create' => 'user/create',
-        'user/account' => 'user/account',
-        'user/login' => 'user/login',
-        'user/logout' => 'user/logout',
-    ),
-);
```

Comment:
I would add this file back in. it won't matter much for the class but if you were doing this for a production site, you'd want some kind of default config file.